diff_controller:
  type: "diff_drive_controller/DiffDriveController"
  left_wheel: 'left_wheel'
  right_wheel: 'right_wheel'
  publish_rate: 20.0 # defaults to 50
  pose_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
  twist_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
  cmd_vel_timeout: 100.0 # we test this separately, give plenty for the other tests
